src/scripts/build-docs.sh

```diff
@@ -4,7 +4,7 @@ if [ -z "$BASH_VERSION" ]; then
     exit 1
 fi
 SCDIR=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
-SCDIR=$(realpath $SCDIR)
-pushd "$SCDIR" > /dev/null || exit
+SCDIR=$(realpath "$SCDIR/../..")
```

Comment:
I still don't understand what this has to do w/ thin executions

Reply:
I'll move this to a separate PR